sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/models/BlobSetTierHeaders.java

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.storage.blob.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+
+/**
+ * Defines headers for SetTier operation.
+ */
+@JacksonXmlRootElement(localName = "Blob-SetTier-Headers")
+public class BlobSetTierHeaders {
+    /*
+     * If a client request id header is sent in the request, this header will
+     * be present in the response with the same value.
+     */
+    @JsonProperty(value = "x-ms-client-request-id")
+    private String clientRequestId;
+
+    /*
+     * This header uniquely identifies the request that was made and can be
+     * used for troubleshooting the request.
+     */
+    @JsonProperty(value = "x-ms-request-id")
+    private String requestId;
+
+    /*
+     * Indicates the version of the Blob service used to execute the request.
+     * This header is returned for requests made against version 2009-09-19 and
+     * newer.
+     */
+    @JsonProperty(value = "x-ms-version")
+    private String version;
+
+    /**
+     * Get the clientRequestId property: If a client request id header is sent
+     * in the request, this header will be present in the response with the
+     * same value.
+     *
+     * @return the clientRequestId value.
+     */
```

Comment:
Linking the property Javadoc instead of writing it all over again.
```suggestion
    /**
     * Get the {@link BlobSetTierHeaders#clientRequestId} property.
     *
     * @return the clientRequestId value.
     */
```

Reply:
We can apply this to all our models and classes really :)